reports/hier/usr/lib/python3/dist-packages/tests/test_reports.py

```diff
@@ -186,6 +192,598 @@ def create_previous_day_table(table_name="sessions", days=1):
         earliest_date = new_date
         days = days - 1
 
+Sql_inject_getDataForReportEntry_parameters = ["ReportEntry", "startDate", "endDate", "extraSelects", "extraConditions", "fromtype", "limit"]
+
+Sql_field_condition_injects = overrides.get(
+    "Sql_field_condition_injects", {
+    "column": [{
+        "javaClass": "com.untangle.app.reports.SqlCondition",
+        "autoFormatValue": False, 
+        # "column": "bypassed is true; {inject};",
+        "column": "bypassed is true {inject}",
+        "operator": "is",
+        "value": "true",
+        "table": None
+    }],
+    # "op": [{
```

Comment:
This may be used in the future ?
That's my assumption when I see commented code.